gnomad_qc/v4/create_release/validate_and_export_vcf.py

```diff
@@ -666,7 +663,7 @@ def populate_info_dict(
     :param subset_pops: Dict of sample global genetic ancestry group names to use for all subsets in `subset_list` unless the subset
         is 'gnomad', in that case `gnomad_pops` is used. Default is POPS.
     :param gnomad_pops: Dict of sample global genetic ancestry group names for gnomAD data type. Default is POPS.
-    :param faf_pops: Dict with faf genentic ancestry group names (keys) and descriptions (values).  Default is FAF_POPS.
+    :param faf_pops: Dict with gnomAD version (keys) and faf genentic ancestry group names(values).  Default is FAF_POPS.
```

Comment:
won't make a functional difference here, but for "populate_subset_info_dict", should the default  for 'faf_pops: Dict[str, str] = FAF_POPS' be changed to FAF_POPS["v4"] so that the types match?

Reply:
Goodp oint, I'm going to change where the FAF_POPS get processed, doesnt make sense to do it here since only subset uses it.